package.json

```diff
@@ -22,12 +22,17 @@
   "scripts": {
     "start": "grunt --color --timing",
     "dist": "grunt dist --color",
+    "prebuild": "grunt prebuild:dist --color",
+    "build": "grunt build:dist --color",
+    "dist:teamcity": "npm run lint && npm run test:coverage:teamcity && grunt css:dist --color && grunt docs --color && grunt webpack",
     "// JAVASCRIPT": "",
     "webpack": "SET \"NODE_ENV=production\" && webpack --progress --colors --display-error-details",
     "// TESTS": "",
     "test": "mocha src/**/*.test.js test/**/*.test.js --require babel-core/register --require mocha-clean --require test/_setup.js --color",
+    "test:teamcity": "mocha --reporter mocha-teamcity-reporter src/**/*.test.js test/**/*.test.js --require babel-core/register --require mocha-clean --require test/_setup.js --color",
```

Comment:
We've got a moch grunt task for this so we should move these mocha build targets into there instead